avocado/core/test.py

```diff
@@ -691,6 +694,30 @@ def fetch_asset(self, name, asset_hash=None, algorithm='sha1',
         return asset.Asset(name, asset_hash, algorithm, locations,
                            self.cache_dirs, expire).fetch()
 
+    @staticmethod
+    def skipIf(condition, reason):
```

Comment:
Yes, I was not sure about this location. Problem is that a common avocado usage starts with `from avocado import Test`. While in unittest world, the common usage is `import unittest`. Using `import avocado` is less common, so users will need to add the `from avocado import skipIf` or stick to `import avocado`. Anyway, you're right. Let's not attach this to the Test class itself. Thank you.